sklearn/utils/testing.py

```diff
@@ -47,6 +54,8 @@
 from numpy.testing import assert_array_less
 import numpy as np
 
+from contextlib import contextmanager
```

Comment:
Please install a pyflakes-based linter in your code editor to spot such useless imports and other potential bugs. If you use atom I would recommend the `linter` and `linter-flake8` packages.